doc/todo/RELEASE.md

```diff
@@ -16,20 +16,21 @@ If no release critical problems are found during testing, continue.
 - Run scripts/dev/update-infos-status with arguments for heuristics (--auto) and check
   transitions in plugin status: especially from/to experimental
 - Write release notes (without hash sums, guid or proper download links)
-- cp doc/todo/NEWS.md doc/news/_preparation_next_release.md
+- cp doc/todo/NEWS.md doc/news/\_preparation_next_release.md
```

Comment:
I would have some suggestions for this file, but it's probably too much to leave as comments here.
May I simply push my suggestions to your branch and merge everything in one go?

Reply:
please go ahead! :)

Reply:
I will leave the style fixes to you then.
https://github.com/ElektraInitiative/libelektra/pull/3776/commits/385bd0cbf587852fcdca3236256d60222f020c5d